Sources/PointFree/LaunchSignup/LaunchSignup.swift

```diff
@@ -36,14 +37,24 @@ private func airtableStuff<I>(_ conn: Conn<I, String>) -> IO<Conn<I, Either<Prel
 func notifyUsOfNewSignup<I>(_ conn: Conn<I, String>) -> IO<Conn<I, String>> {
   return IO {
 
-    // Fire-and-forget to notify us that someone signed up
-    parallel(
-      sendEmail(
+    // Fire-and-forget to send some emails
+    zip(
```

Comment:
Are you sure this is parallelizing? I think `run` itself is synchronous, and that we need to do something like:

``` swift
let both = tuple
  <^> parallel <| sendEmail(…)
  <*> parallel <| sendEmail(…)
both.run { _ in }
```

Reply:
(This'll hopefully improve when we rethink some of our IO story.)

Reply:
Well there’s two runs! One for EitherIO which just gets the underlying IO and then I run it as parallel. So I do think this is parallel

Reply:
Oh nm missed which `zip` was being used!